zucchini/gradescope.py

```diff
@@ -80,6 +86,19 @@ def _two_decimals(grade, frac):
         """Convert a fraction to string with two decimal points"""
         return '{:.02f}'.format(grade.to_float(frac))
 
+    @staticmethod
+    def _get_status(part_score):
+        """
+        Override how Gradescope determines status. If a student gets 0% of 0
+        points, we send them 0/0, which it cannot distinguish from 100% of 0
+        points, which is 0/0 too. So use the point percentage to calculate
+        this, not points earned.
+
+        tl;dr: You can fail a test worth 0 points, but this blows Arjun's mind
+        """
+        return GradescopeAutograderTestOutput.STATUS_PASSED if part_score > 0 \
```

Comment:
I wonder if this should be `< 1` actually. Not sure, kind of subjective

Reply:
Not sure if < 1 is a fraction or a point, but > 0 should be fine.

Reply:
God damn